### ISSUE TITLE
Inference Colab file of ARDM

### ISSUE THREAD

**Issue by Aditya6500** [qywu/ARDM#6]
I am faces few issues in ARDM Inference file of colab
I am getting error while using
model_A.load_state_dict(model_A_states)
model_B.load_state_dict(model_B_states).
Its unable to match all the keys.
I tried adding strict = False

model_A.load_state_dict(model_A_states,strict =False)
model_B.load_state_dict(model_B_states , strict = False)
The code runs but few keys don't match.
On proceeding in this manner I am facing error here
logits, past = model_A(prev_input, past=past)
error : past argument not supported. I tried removing past argument 
but it gives error for the next line of code.
logits = logits[:, -1, :] / temperature

Can you please help me fix this code.

**Comment by qywu** [comment]
You might need to use a older version of transformers to load the weights correctly.

**Comment by Aditya6500** [comment]
Thank you so much , Version 2.2.2 worked.
